packages/trifid/instances/tbbt/config.yaml

```
@@ -30,6 +30,8 @@ plugins:
     paths:
       - /graph-explorer
       - /graph-explorer/
+    config:
+      title: "The Big Bang Theory"
```

For that demo instance, could you include Graph Explorer explicitly in the title as well, like this?

```suggestion
      title: "The Big Bang Theory - Graph Explorer"
```